content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/430-nextjs-prisma-client-dynamic.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Best practices for using Prisma Client with Next.js and Vercel caching'
```

Comment:
Is "Vercel caching" the correct term for this? I don't know how Vercel and Next.js itself talks about this.

Reply:
Thanks for the comment. I dug more into this and it looks like this is a Next.JS feature, [Data Cache](https://nextjs.org/docs/app/building-your-application/caching#data-cache)

I'll update this page more correctly referencing this feature.